package.json

```diff
@@ -16,12 +16,13 @@
     "dist": "gulp dist"
   },
   "devDependencies": {
+    "babelify": "^6.1.3",
     "browserify": "^10.2.4",
     "chai": "^3.0.0",
     "del": "^1.2.0",
     "envify": "~3.4.0",
-    "es6ify": "~1.6.0",
     "gulp": "^3.9.0",
+    "gulp-es6-module-to-closure": "^1.0.7",
```

Comment:
Nice that this is A Thing.

Would be even nicer if it used goog.module, but alright.